extensions/amp-link-rewriter/amp-link-rewriter.md

```diff
@@ -0,0 +1,218 @@
+<!--
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# `amp-link-rewriter`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Run link-rewriter inside your AMP page</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-link-rewriter" src="https://cdn.ampproject.org/v0/amp-link-rewriter-0.1.js`>&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>nodisplay</td>
+  </tr>
+</table>
+
+## Overview
+
+`amp-link-rewriter` allows publishers to rewrite URL based on configurable pattern.
+
+## Getting started
+
+
+**Add the required script**
+Inside the `<head>...</head>` section of your AMP page, insert this code before the line `<script async src="https://cdn.ampproject.org/v0.js"></script>`
+
+Code:
+```html
+    <script async custom-element="amp-link-rewriter" src="https://cdn.ampproject.org/v0/amp-link-rewriter-0.1.js`"></script>
+```
+
+**Add the amp-link-rewriter extension**
+Inside the `<body>...</body>` section of your AMP page, insert this code:
```

Comment:
Can we phrase it like below is an example. Asking because we want to the extension to be more generic. And could you please document the supported config params? Thank you